dist/ember-charts.js

```diff
@@ -5304,27 +5304,50 @@ define('ember-charts/mixins/pie-legend', ['exports', 'module', 'ember'], functio
       // cause labelTop to exceed the space allocated for the legend.
       var labelTop = 0;
 
-      var arr = labelStrings.slice(1).length;
-      for (var i = 0, len = arr.length; i < len; i++) {
-        nextLabel = arr[i];
-        currentText = row.text();
-        row.text("" + currentText + ", " + nextLabel);
-        rowNode = row.node();
-        if (rowNode.getBBox().width > this.get('legendWidth')) {
-          if (labelTop + rowNode.getBBox().height > this.get('maxLabelHeight')) {
-            row.text("" + currentText + ", ...");
-            break;
-          } else {
-            row.text("" + currentText + ",");
-            labelTop += rowNode.getBBox().height;
-            row = labels.append('text').text(nextLabel).attr(this.get('legendLabelAttrs')).attr('dy', labelTop);
+      var remainingLabelStrings = labelStrings.slice(1);
```

Comment:
The weird thing is that it doesn't do that if the array isn't stored in a variable. 

```
const array = [1, 2, 3];

for (var x of array) {
  console.log(x);
}
```
compiles to something very different than 
```
for (var x of [1, 2, 3]) {
  console.log(x);
}
```